### PR TITLE
Clearify users and redistribution

### PR DESCRIPTION
**Pull request by SchrodingerZhu**
This patch addresses some concerns by third-party mirrors:

- we explicitly waive the liability and warranty of the repository
- change the policy to forbid redistribution of the repo in other os distributions, instead of "forbidding" the use of certain group of users.

### CODE REVIEW

File: src/content/docs/policy/repository_policy.md [hunk 1]
```diff
@@ -6,47 +6,79 @@ CachyOS Repository Usage Policy
...
+## 5. Redistribution of the Repository
+
+By "redistribution", we mean the behaviors of inclusion of the CachyOS repository (and its mirrors) or packages obtained from the CachyOS repository as a part of the distributed image of the operating system. 
+This also applies to the behaviors of providing the utilities that enable CachyOS repository by users' choice, or providing any distributed or official document that guide users to enable CachyOS 
+repository (and its mirrors) by their means. 
+
+The CachyOS repository is exclusively authorized for redistribution in the following distributions:
+
+- CachyOS
+- ArchLinux
```
Comment (vnepogodin) [inline src/content/docs/policy/repository_policy.md]: I don't think we should put ArchLinux here

**Review by vnepogodin — CHANGES_REQUESTED** [review]

**Comment by vnepogodin** [comment]
seems good overall

File: src/content/docs/policy/repository_policy.md (hunk 1 above)
Comment (SchrodingerZhu) [reply src/content/docs/policy/repository_policy.md]: fixed

**Review by vnepogodin — APPROVED** [review]
many thanks. looks good to me

**Review by Aarrayy — APPROVED** [review]
Thanks for the PR! everything looks good

**Comment by ptr1337** [comment]
Thank you for updating this, also fine from my side.

**Review by ptr1337 — APPROVED** [review]

**Pull request merged**